presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/Partition.java

```diff
@@ -145,6 +179,20 @@ public long getLastDataCommitTime()
         return lastDataCommitTime;
     }
 
+    /**
+     * A warehouse-wide unique identifier for a specific version of a
+     * specific partition in a specific table.
+     */
+    @JsonProperty
```

Comment:
Does @JsonProperty work well with Optional?

Reply:
It should as long as the JDK8 module is installed in the mapper. Tests for this pass. We could use null instead though I'm told that Optional is preferred in Presto. Note that partitionVersion is already declared Optional and is an `@JsonProperty` in this class so I think this is safe.